app/javascript/store/modules/froggo_team/actions.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
```

Comment:
Definition for rule 'vue/max-len' was not found    vue/max-len

Reply:
no entiendo este error :( es de mi código?